src/main/java/com/homework/lesson12/MyTubeParser.java

```diff
@@ -0,0 +1,71 @@
+package com.homework.lesson12;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.CommentThreadListResponse;
+import com.google.api.services.youtube.model.VideoListResponse;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import static com.homework.lesson12.MyTube.getService;
+
+@ToString
+@Getter
+
+public class MyTubeParser {
+    private static final String DEVELOPER_KEY = "src/main/resources/dev.txt";
+    private String videoId = "BiVLUgJ5UdY";
+    private String videoTitle;
+    private List<Comment> commentsList = new ArrayList<>();
+
+    public void videoTitleParsing() throws IOException, GeneralSecurityException {
```

Comment:
This method is too long. Please break it into parts. You could use for each part private method.